frontend/packages/operator-lifecycle-manager/src/components/descriptors/status/index.spec.tsx

```diff
@@ -89,6 +89,14 @@ describe('Status descriptors', () => {
     expect(wrapper.find('dd').text()).toEqual('example.com');
   });
 
+  it('renders an external link status', () => {
+    descriptor['x-descriptors'] = [StatusCapability.w3ExternalLink];
+    descriptor.path = 'externalLink';
+    wrapper = wrapper.setProps({ descriptor });
+
+    expect(wrapper.find('dd').text()).toEqual('example.com');
```

Comment:
You'll need to update this to look for the full URL with scheme once you've made the other updates I suggested.